catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -168,18 +166,20 @@ public void configure() throws Exception {
                 }
                 LOGGER.debug("inbox = {}", inbox);
 
-                RouteDefinition routeDefinition = from(inbox);
+                StringBuilder attributeOverrideString = new StringBuilder();
 
                 if (parameters != null) {
-                    for (String parameter : parameters) {
-                        String[] parameters = parameter.split("=");
-                        if (parameters.length == 2) {
-                            routeDefinition.setHeader(parameters[0], simple(parameters[1]));
-                            LOGGER.debug("Adding Header to RouteDefinition : {} {}", parameters[0], parameters[1]);
+                    parameters.forEach(parameter -> {
```

Comment:
I agree with trimming the parameters, so `String.join(",", parameters);` might not be best here